doc/source/building/distutils_equivalents.rst

```diff
@@ -0,0 +1,28 @@
+.. _distutils-meson-equivalents:
+
+Meson and ``distutils`` ways of doing things
+--------------------------------------------
+
+*Old workflows (numpy.distutils based):*
+
+1. ``python runtests.py``
+2. ``python setup.py build_ext -i`` + ``export
+   PYTHONPATH=/home/username/path/to/numpy/reporoot`` (and then edit pure
+   Python code in NumPy and run it with ``python some_script.py``).
+3. ``python setup.py develop`` - this is similar to (2), except in-place build
+   is made permanently visible in env.
+4. ``python setup.py bdist_wheel`` + ``pip install dist/numpy*.whl`` - build
+   wheel in current env and install it.
+5. ``pip install .`` - build wheel in an isolated build env against deps in
+   ``pyproject.toml`` and install it. *Note: be careful, this is usually not
+   the correct command for development installs - typically you want to use (4)
+   or* ``pip install . -v --no-build-isolation``.
+
+*New workflows (Meson and meson-python based):*
```

Comment:
Maybe this should come first, before the setuptools/distutils incantations? Since this is what we want people to use going forward.

Reply:
The thinking was that this page is only useful for people migration from old to new, hence old should come first. I don't think there is another purpose - this page can simply be deleted once we're all used to the new commands.